CHANGELOG.md

```diff
@@ -1,99 +1,10 @@
 ﻿# Changelog
 
-## v1.0.0
+## v1.0.1
 
-- Added `SuspendInstanceAsync` and `ResumeInstanceAsync` to `DurableTaskClient`.
-- Rename `DurableTaskClient` methods
-    - `TerminateAsync` -> `TerminateInstanceAsync`
-    - `PurgeInstanceMetadataAsync` -> `PurgeInstanceAsync`
-    - `PurgeInstances` -> `PurgeAllInstancesAsync`
-    - `GetInstanceMetadataAsync` -> `GetInstanceAsync`
-    - `GetInstances` -> `GetAllInstancesAsync`
-- `TaskOrchestrationContext.CreateReplaySafeLogger` now creates `ILogger` directly (as opposed to wrapping an existing `ILogger`).
-- Durable Functions class-based syntax now resolves `ITaskActivity` instances from `IServiceProvider`, if available there.
-- `DurableTaskClient` methods have been touched up to ensure `CancellationToken` is included, as well as is the last parameter.
-- Removed obsolete/unimplemented local lambda activity calls from `TaskOrchestrationContext`
-- Input is now an optional parameter on `TaskOrchestrationContext.ContinueAsNew`
-- Multi-target gRPC projects to now use `Grpc.Net.Client` when appropriate (.NET6.0 and up)
+### Microsoft.DurableTask.Client
 
-*Note: `Microsoft.DurableTask.Generators` is remaining as `preview.1`.*
-
-## v1.0.0-rc.1

```

Comment:
I assume you're removing the pre-release notes since we're now post-GA?

Reply:
Yeah. I think we can rely on a practice of moving release notes into GitHub released with each release to act as the larger by-release view. I am not sure if we even need to keep `CHANELOG.md` as this PR has us update `RELEASENOTES.md` on a per-project basis.